root/relationship/linkattributetype/form.tt

```diff
@@ -1,26 +1,26 @@
 <form action="[% c.req.uri %]" method="post">
     [%- USE r = FormRenderer(form) -%]
 
-    [% form_row_select(r, 'parent_id', add_colon(l('Parent'))) %]
+    [% form_row_select(r, 'parent_id', add_colon('Parent')) %]
 
     [% WRAPPER form_row %]
-        [% r.label('child_order', add_colon(l('Child order'))) %]
+        [% r.label('child_order', add_colon('Child order')) %]
         [% r.number('child_order') %]
         [% field_errors(form, 'child_order') %]
     [% END %]
 
-    [% form_row_text(r, 'name', add_colon(l('Name'))) %]
+    [% form_row_text(r, 'name', add_colon('Name')) %]
 
     [% WRAPPER form_row %]
-        [% r.label('description', add_colon(l('Description'))) %]
+        [% r.label('description', add_colon('Description')) %]
         [% r.textarea('description', { cols => 80, rows => 6 }) %]
         [% field_errors(form, 'description') %]
     [% END %]
 
-    [% form_row_checkbox(r, 'creditable', l('This attribute supports free text credits')) %]
-    [% form_row_checkbox(r, 'free_text', l('This attribute uses free text values')) %]
+    [% form_row_checkbox(r, 'creditable', 'This attribute supports free text credits') %]
+    [% form_row_checkbox(r, 'free_text', 'This attribute uses free text values') %]
```

Comment:
I would keep these translated in `futureColumnLabels` as it is likely to be reused for link attribute type documentation.

Reply:
Actually, this is not needed because the same strings are already in `root/relationship/linkattributetype/RelationshipAttributeTypeIndex.js` so they will remain available.